examples/minimal.yml

```diff
@@ -0,0 +1,24 @@
+kernel:
+  image: "mobylinux/kernel:4.9.x"
+  cmdline: "console=ttyS0 console=tty0 page_poison=1"
+init:
+  - linuxkit/init:42fe8cb1508b3afed39eb89821906e3cc7a70551
+  - mobylinux/runc:b0fb122e10dbb7e4e45115177a61a3f8d68c19a9
```

Comment:
can we use `linuxkit/runc:b0fb122e10dbb7e4e45115177a61a3f8d68c19a9`?  Seems to exist on hub: https://hub.docker.com/r/linuxkit/runc/tags/